docs/source/tutorials/sklearn_svc.ipynb

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
<p>we use the SKLearn library to implement SVC in the following</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
fixed!